cmd/celvet/main.go

```diff
@@ -81,23 +91,55 @@ func main() {
 	}
 
 	limitErrors := celvet.CheckMaxLimits(structural)
+	costErrors, compileErrors, otherErrors := celvet.CheckExprCost(structural)
```

Comment:
This trio of error slices is used quite a bit in the code, consider introducing a type for it?  Type could also have a `hasErrors` function that returns true if any of the three slices is len > 0.

Reply:
Fixed - I've added a `LinterErrors` struct that's used throughout the codebase now.